applications/GeoMechanicsApplication/custom_processes/calculate_incremental_displacement_process.cpp

```diff
@@ -0,0 +1,41 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors: Aron Noordam,
+//
+#include "calculate_incremental_displacement_process.h"
+#include "includes/element.h"
+#include "includes/kratos_flags.h"
+#include "includes/kratos_parameters.h"
+#include "includes/model_part.h"
+#include "utilities/variable_utils.h"
+#include "geo_mechanics_application_variables.h"
+
+namespace Kratos
+{
+
+    CalculateIncrementalDisplacementProcess::CalculateIncrementalDisplacementProcess(ModelPart& rModelPart, const Parameters& rProcessSettings)
+    : Process(Flags()), mrModelPart{rModelPart}
+{
+}
+
+void CalculateIncrementalDisplacementProcess::Execute()
+{
+    KRATOS_TRY
+
+        block_for_each(mrModelPart.Nodes(), [&](Node& rNode) {
```

Comment:
Please note that the lambda expression now captures any local variables of the enclosing function (due to using `[&]` as the capture list). Prefer to only capture what you need. Since that is nothing in this case, a capture list of `[]` will do.
```suggestion
        block_for_each(mrModelPart.Nodes(), [](Node& rNode) {
```

Note that the suggested change fixes a code smell found by SonarQube.